packages/indexer-cli/src/commands/indexer/actions/get.ts

```diff
@@ -27,9 +27,42 @@ ${chalk.dim('Options:')}
       --orderBy id|deploymentID|amount|priority|...|updatedAt       Order actions by a specific field (default: id)
       --orderDirection asc|desc                                     Order direction (default: desc)
       --first [N]                                                   Fetch only the N first records (default: all records)
+      --fields [field1,field2]                                      Comma-separated names of the fields to display
   -o, --output table|json|yaml                                      Choose the output format: table (default), JSON, or YAML
 `
 
+const actionFields: (keyof Action)[] = [
```

Comment:
I'm having difficulty figuring out how to access the property names from the `Action` class.

I have tried using `Model.rawAttributes`, [as shown in `sequelize` docs](https://sequelize.org/docs/v6/other-topics/typescript/#getting-a-models-attributes), but all I got was `undefined` instead of a property list.
```ts
export function getAttributeMetadata<M extends Model>(model: ModelStatic<M>): string[] {
  return Object.keys(model.rawAttributes)
}

console.log(getAttributeMetadata(Action))
> undefined
```

I also tried calling `Action.getAttributes`, [as we already do in `indexer-common`](https://github.com/graphprotocol/indexer/blob/main/packages/indexer-common/src/indexer-management/__tests__/resolvers/actions.ts#L144), and this time I got an empty array as a result.
```ts
function extractActionKeys(): string[] {
    const result = []
    for (const actionKey in Action.getAttributes()) {
        result.push(actionKey)
    }
    return result
}
console.log(extractActionKeys())
> []
```

I'm not sure what I'm missing here.

Reply:
Would it be OK to perform this validation after we got the query results?
I believe that this should work when dealing with `Action` instances instead of the `Action` class.

Reply:
Yeah that sounds good! I'm ok with leaving in the hardcoded array for now though if that would hold this up.